src/backends/netlify-auth/implementation.js

```diff
@@ -27,7 +27,7 @@ export default class NetlifyAuth extends GitHubBackend {
     AuthenticationPage.authClient = this.authClient;
   }
 
-  setUser() {
+  restoreUser() {
     const user = this.authClient.currentUser();
     if (!user) return Promise.reject();
     return this.authenticate(user);
```

Comment:
Note: The result of what this function used to return was the `stored` value, now it would be the `this.authenticate` return value. I think this is what was originally intended, but not actually done, can someone else verify?